package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "mobx",
-  "version": "3.1.5",
+  "version": "3.1.6",
   "description": "Simple, scalable state management.",
   "main": "lib/mobx.js",
+  "module": "lib/es2015/mobx.js",
+  "jsnext:main": "lib/es2015/mobx.js",
```

Comment:
I know it's just one extra line, but IMHO there's no reason to promote this proprietary, obsolete package.json property.

This only leads to people getting confused.